src/sidebar/services/groups.ts

```diff
@@ -478,4 +491,50 @@ export class GroupsService {
       userid: 'me',
     });
   }
+
+  /**
+   * Fetch members for focused group from the API and load them into the store.
+   */
+  private async _loadFocusedGroupMembers(groupId: string): Promise<void> {
+    // Open groups do not have members, so we can save the API calls
```

Comment:
All group types can have members in the data model. There isn't a way for the owner of an open group to invite members to join yet, but the intent is that will be added.

Reply:
Good. Then I'll simplify it so that the call is always done. If an empty list is returned for whatever reason, so be it.